README.md

```diff
@@ -29,6 +29,18 @@ resolc --version
 
 `revive` requires a build of LLVM 18.1.4 or later including `compiler-rt`. Use the provided [build-llvm.sh](build-llvm.sh) build script to compile a compatible LLVM build locally in `$PWD/llvm18.0` (don't forget to add that to `$PATH` afterwards). 
 
+### Cross-compilation to WASM
+
+Install [emscripten](https://emscripten.org/docs/getting_started/downloads.html)
+
+```bash
+export EMSDK_ROOT=<PATH_TO_EMSCRIPTEN_SDK>
+bash emscripten-build-llvm.sh
+source $EMSDK_ROOT/emsdk_env.sh
+export LLVM_LINK_PREFIX=${PWD}/llvm18.0-emscripten
+make install-wasm
+```
```

Comment:
This is not complete yet, I tried to build this branch locally on mac OS however the `builtins` and `stdlib` crates `build.rs` fail.

Reply:
On my mac it is working, could you please share the error message

Reply:
The `build.rs` scripts of several crates rely on the LLVM (bin) install dir being added to `$PATH`, e.g. `llvm-config` the `llvm-as` assembler and so on.

If you leave them in `$PATH` it'll accidentially work :) I suggest either pointing out that the native LLVM build is required anyways (see `Installation`), or adding `bash build-llvm.sh && 
export PATH=${PWD}/llvm18.0/bin:$PATH` here.

Reply:
Does the standard path :
```
bash build-llvm.sh &&  export PATH=${PWD}/llvm18.0/bin:$PATH
make install-bin
```
 work for you?
 On my pc it does not compile and requires full llvm release

Reply:
`make install-bin` isn't necessary, we just need the LLVM build. I just tested it, works on my machine (TM). Might be a good idea to reproduce it on CI though :)